features/main/embedded.feature

```diff
@@ -0,0 +1,108 @@
+Feature: Embedded resources support
+  In order to use a hypermedia API
+  As a client software developer
+  I need to be able to retrieve embedded resources only
+
+  @dropSchema
+  Scenario: Do nothing
```

Comment:
Temporary hack to allow the single `./vendor/bin/behat --stop-on-failure -v --name="Embedded resources support"`